src/app/shared/components/icon-button/icon-button.component.ts

```diff
@@ -0,0 +1,57 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {IconProp, SizeProp} from "@fortawesome/fontawesome-svg-core";
+import {IconButton, IconButtonSize} from "../../enums/icon-button";
+import {faTrashCan, faXmark} from "@fortawesome/free-solid-svg-icons";
+
+@Component({
+  selector: 'sofie-icon-button',
+  templateUrl: './icon-button.component.html',
+  styleUrls: ['./icon-button.component.scss']
+})
+export class IconButtonComponent implements OnInit {
+
+  @Input()
+  public iconButton: IconButton
+
+  @Input()
+  public iconButtonSize?: IconButtonSize
+
+  public iconButtonProp: IconProp
+  public iconButtonSizeProp: SizeProp
+
+  constructor() {}
+
+  ngOnInit(): void {
+    this.iconButtonProp = this.getIconProperty(this.iconButton)
+    this.iconButtonSizeProp = this.getIconSizeProperty(this.iconButtonSize)
+  }
+
+  public getIconProperty(iconButton: IconButton): IconProp {
+    switch (iconButton) {
+      case IconButton.XMARK:
+        return faXmark
+      case IconButton.DELETE:
+        return faTrashCan
+      default:
+        return faXmark
+    }
+  }
+
+  public getIconSizeProperty(iconButtonSize?: IconButtonSize): SizeProp {
+    switch (iconButtonSize) {
+      case IconButtonSize.XS:
+        return "xs"
+      case IconButtonSize.SMALL:
+        return "sm"
+      case IconButtonSize.LARGE:
+        return "lg"
+      case IconButtonSize.XL:
+        return "xl"
+      case IconButtonSize.TWO_XL:
```

Comment:
Is there a reason this is TWO_XL and not just `XXL`? :D

Reply:
To piggyback on this. What is the reason for using both abbreviations and the fully written word i.e. you are using both `XS` and `SMALL`.
You should choose one and commit to it, so either you do `XS` and `S` or you do `EXTRA_SMALL` and `SMALL`

Reply:
I believe i lost myself a bit in this. I for some reason made it fit the actual values that you pass to the size property, don't ask me why :D. I'll fix this in the next commit.

Reply:
I've changed it to be  `XS`, `S`, `M` and so forth, instead.